values.yaml

```diff
@@ -484,7 +484,7 @@ server:
   # This can also be set using Helm's `--set` flag using the following syntax:
   #
   # ```shell
-  # --set 'server.extraConfig="{"log_level": "DEBUG"}"'
+  # --set 'server.extraConfig='{"log_level": "DEBUG"}"'
```

Comment:
This single quote would be an extra one. I think this example needs double quotes inside the json string escaped.

Reply:
good catch, thx!

Reply:
This does not seem resolved 🤔